data/json/mapgen/lab/lab_surface/lab_surface_big_z2.json

```diff
@@ -0,0 +1,243 @@
+        ],
+        "L": { "item": "cleaning", "chance": 70, "repeat": [ 1, 8 ] },
+        "t": [ { "item": "snacks", "chance": 5 }, { "item": "dining", "chance": 10 } ]
+      },
+      "palettes": [ "lab_upstairs_palette" ],
+      "place_nested": [
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 45, 47 ] },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 48, 57 ], "repeat": 6 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 60, 69 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 72, 81 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 24, "y": [ 84, 93 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 30, "y": [ 36, 45 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 30, "y": [ 48, 57 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 30, "y": [ 59, 69 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 30, "y": [ 72, 81 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 89, "y": [ 45, 47 ] },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 89, "y": [ 48, 57 ], "repeat": 6 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 89, "y": [ 72, 81 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 45, 47 ] },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 48, 57 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 59, 69 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 72, 81 ], "repeat": 7 },
+        { "chunks": [ "surface_sub_hall_clutter" ], "x": 95, "y": [ 84, 93 ], "repeat": 7 },
+        { "chunks": [ "lab_surface_office_5x5" ], "x": 12, "y": 36 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 18, "y": 42 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 12, "y": 42 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 48 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 60 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 72 },
+        { "chunks": [ "lab_surface_workspace_Edoor_11x11" ], "x": 12, "y": 84 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 36 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 48 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 60 },
+        { "chunks": [ "lab_surface_workspace_Wdoor_11x11" ], "x": 32, "y": 72 },
+        { "chunks": [ "lab_surface_2partworkspace_front_5x5" ], "x": 18, "y": 96 },
+        { "chunks": [ "lab_surface_2partworkspace_back_5x5" ], "x": 12, "y": 96 },
+        { "chunks": [ "lab_surface_office_5x5" ], "x": 103, "y": 36 },
```

Comment:
I don't know if you want to do anything about it, but in playtesting the `5x5 three-seat office` variation spawned here (`"x": 103, "y": 36 `), which plopped a bookcase right in front of the door.

![image](https://user-images.githubusercontent.com/11464/54486755-800e3680-4841-11e9-836e-f5a035bddcb3.png)

Reply:
It appears I misunderstood the role of the `"rotate"` flag in nested mapgen. Resetting to 0,3 that should fix this and similar problems with blocked doors.